<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UForm_fillSubsDbfSpojyvannyaCO</class>
 <widget class="QWidget" name="UForm_fillSubsDbfSpojyvannyaCO">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Заповнення dbf файлу фактичного споживання для Відділу субсидії</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="margin">
    <number>2</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>1. Вибір dbf файлу</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLineEdit" name="lineEdit_filePath"/>
        </item>
        <item>
         <widget class="QToolButton" name="toolButton_browse">
          <property name="text">
           <string>Пошук файлу</string>
          </property>
          <property name="icon">
           <iconset resource="tke/tke_rc">
            <normaloff>:/pictures/pictures/kghostview.png</normaloff>:/pictures/pictures/kghostview.png</iconset>
          </property>
          <property name="toolButtonStyle">
           <enum>Qt::ToolButtonTextBesideIcon</enum>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>2. Заповнення файлу</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QPushButton" name="pushButton_fillFile">
        <property name="text">
         <string>Заповнити файл</string>
        </property>
        <property name="icon">
         <iconset resource="tke/tke_rc">
          <normaloff>:/pictures/pictures/editors_section.png</normaloff>:/pictures/pictures/editors_section.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>3. Результати виконання</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <widget class="QTableWidget" name="tableWidget">
        <column>
         <property name="text">
          <string>Iter</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Дата</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Рахунок</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Факт. викор. ГКал., N2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Тариф за м2, P2</string>
         </property>
        </column>
        <column>
         <property name="text">
          <string>Результат</string>
         </property>
        </column>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="tke/tke_rc"/>
 </resources>
 <connections/>
</ui>
